client/components/domains/free-domain-explainer/index.jsx

```diff
@@ -45,19 +45,25 @@ class FreeDomainExplainer extends React.Component {
 	}
 
 	render() {
-		const { translate, isReskinned } = this.props;
+		const { translate, isReskinned, locale } = this.props;
 		const { TextWrapper } = this;
 
 		return (
 			<div className="free-domain-explainer card is-compact">
 				<header>
 					<h1 className="free-domain-explainer__title">
-						{ translate( 'Get a free one-year domain registration with any paid plan.' ) }
+						{ locale === 'en'
+							? translate( 'Get a free one-year domain registration with any paid annual plan.' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.